llvm/test/MC/SBF/sbf-jmp.s

```diff
@@ -1,8 +1,13 @@
 # RUN: llvm-mc %s -triple=sbf-solana-solana --mcpu=sbfv2 --show-encoding \
 # RUN:     | FileCheck %s --check-prefix=CHECK-ASM-NEW
+# RUN: llvm-mc %s -triple=sbf-solana-solana --show-encoding \
+# RUN:     | FileCheck %s --check-prefix=CHECK-ASM-OLD
 # RUN: llvm-mc %s -triple=sbf-solana-solana --mcpu=sbfv2 -filetype=obj \
-# RUN:     | llvm-objdump -d -r - \
+# RUN:     | llvm-objdump --mcpu=sbfv2 -d -r - \
```

Comment:
maybe this won't be necessary when you rebase on #77?

Reply:
I'll remove it once I merge that.

Reply:
Thanks